lib/link/users/profile.ex

```diff
@@ -8,6 +8,7 @@ defmodule Link.Users.Profile do
 
   schema "user_profiles" do
     field :fullname, :string
+    field :displayname, :string
```

Comment:
Shall we put this one on `User`? I can imagine we will show it in multiple places? Can also be a change in a following PR.

Reply:
Good idea, will put it there in next PR

Reply:
Maybe not for this Sprint. Since it is now part of the profile update page..